dash/dash.py

```diff
@@ -373,6 +375,7 @@ def __init__(  # pylint: disable=too-many-statements
         long_callback_manager=None,
         background_callback_manager=None,
         add_log_handler=True,
+        hooks: Union[RendererHooks, None] = None,
```

Comment:
I think w should still mention this in the docstring, even though it's primarily meant for internal use. Something like "a dict of javascript functions for modifying behavior of the Dash renderer"

Reply:
Good call, done!